c2corg_api/scripts/migration/create_unique_normalized_usernames.sql

```diff
@@ -108,16 +108,19 @@ CREATE UNIQUE INDEX ix_punbb_users_lower_username_unique
   (lower(username_unique::text) COLLATE pg_catalog."default");
 
 UPDATE punbb_users
-SET username_unique = substring(username_normalized from 1 for  24) || number
+SET
+  username_unique = CASE
+    WHEN number > 1 THEN substring(username_normalized from 1 for  24) || number
+    ELSE username_normalized
```

Comment:
Does this mean that when duplicates are found, the first one is not changed, only the following ones?